lib/opentok.js

```diff
@@ -477,6 +478,168 @@ OpenTok = function (apiKey, apiSecret, env) {
+   *   </li>
+   *
+   * </ul>
+   *
+   * <p>If you don't pass in an <code>options</code> argument,
+   * the method returns up to 1000 renders
+   * starting with the first render recorded.
+   *
+   * @param callback {Function} The function to call upon completing the operation. Two arguments
+   * are passed to the function:
+   *
+   * <ul>
+   *
+   *   <li>
+   *      <code>error</code> &mdash; An error object (if the call to the method fails).
+   *   </li>
+   *
+   *   <li>
+   *       <code>renders</code> &mdash; An array of {@link Render} objects.
+   *   </li>
+   *
+   * </ul>
+   *
+   * @method #listRenders
+   * @memberof OpenTok
+   */
+  this.listRenders = render.listRenders.bind(null, apiConfig);
+
+  /**
+   * Gets an {@link Render} object for the given render ID.
+   *
+   * @param renderId {String} The render ID.
+   *
+   * @param callback {Function} The function to call upon completing the operation. Two arguments
+   * are passed to the function:
+   * <ul>
+   *   <li><code>error</code> &mdash; An error object (if the call to the method fails). </li>
+   *   <li><code>render</code> &mdash; The {@link Render} object.</li>
+   * </ul>
+   *
+   * @method #getArchive
+   * @memberof OpenTok
+   */
+  this.getRender = render.getRender.bind(null, apiConfig);
+
+  /**
+  * Starts rendering an OpenTok session.
+  * <p>
+  * Clients must be actively connected to the OpenTok session for you to successfully start
+  * rendering a session.
+  * <p>
+  *
+  * @param options {Object} An optional options object with the following properties (each
+  * of which is optional):
+  * <p>
+  * <ul>
+  *   <li>
+  *     <code>sessionId</code> (String) &mdash; The ID of a session (generated with the same 
+  *    `APIKEY`as specified in the URL) which you wish to start rendering into
+  *   </li>
+  *   <li>
+  *     <code>token</code> (String) &mdash; A valid OpenTok token with a Publisher role and 
+  *     (optionally) connection data to be associated with the output stream.
+  *   </li>
+  *   
+  *   <li>
+  *     <code>url</code> (String) &mdash; A publically reachable URL controlled by the 
+  *     customer and capable of generating the content to be rendered without user intervention.
+  *   </li>
+  *   
+  *   <li>
+  *     <code>properties</code> (Object) &mdash; Initial configuration of Publisher properties for 
+  *     the composed output stream.
+  *   </li>
+  *   
+  *   <li>
+  *     <code>maxDuration</code> (Number) &mdash; The maximum time allowed for the Render, in 
+  *     seconds. After this time, the Render will be stopped automatically, if it is still running.
+  *   </li>
+  *   
+  *   <li>
+  *     <code>resolution</code> (String) &mdash; Resolution of the display area for the composition.
+  *   </li>
+  *   
+  *   <li>
+  *     <code>statusCallbackUrl</code> (String) &mdash; URL of the customer service where the 
+  *     callbacks will be received.
+  *   </li>
```

Comment:
We are not documenting this REST API option. Developers should use account portal to register callbacks. See https://tokbox.com/developer/guides/experience-composer/#configuring-callbacks.